.circleci/config.yml

```
@@ -135,6 +135,11 @@ jobs:
           environment:
             SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD: 1
           command: ./gradlew :libs:WordPressProcessors:test --stacktrace
+      - run:
+          name: Test ImageEditor
+          environment:
+            SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD: 1
+          command: ./gradlew :libs:ImageEditor:test --stacktrace
```

@malinajirka It looks like you're missing the `image-editor` module in the command you used, that's why it's failing with `Project 'ImageEditor' not found in project ':libs'.` error. This should fix it:

```suggestion
          command: ./gradlew :libs:image-editor:ImageEditor:test --stacktrace
```

---

I was using the correct name before and I unintentionally changed this right before the commit 😞. Thanks!